docs/framework/deployment/index.md

```diff
@@ -0,0 +1,18 @@
+# Installing the .NET Framework on Windows
+
+The .NET Framework comes with Windows. There are multiple versions of the .NET Framework, however, each version of Windows only includes one version. If you would like to use a different version of the .NET Framework for a given version of Windows, then you must install it.
```

Comment:
That's not true. .NET Framework has come with Windows since (variants of) XP and with all Windows SKUs since Vista.

I'll address your other points.